frontends/mit-open/src/pages/DashboardPage/UserListDetailsTab.tsx

```diff
@@ -28,6 +31,8 @@ const UserListDetailsTab: React.FC<UserListDetailsTabProps> = (props) => {
       items={items}
       isLoading={itemsQuery.isLoading}
       isFetching={itemsQuery.isFetching}
+      showSort={!!user?.is_authenticated}
```

Comment:
Redundant as this page won't be routable for users who are not signed in, so just for good measure and to show intent.

Reply:
BTW: Userlists have a privacy setting `"privacy_level": "private" | "unlisted", where "unlisted" is meant to be. "shareable, but doesn't show up in our search index".

So it's possible this isn't completely redundant. Though I would expect we'd have a separate page for sharing a userlist (not the dashboard), and this feature is more theoretical at the moment than real.